scripts/post-react.js

```diff
@@ -0,0 +1,10 @@
+const gulp = require("gulp");
+const replace = require("gulp-replace");
+
+gulp.start('examples_react', function() {
+	return gulp.src('./dist/examples/react/**/*.html')
+		.pipe(replace('/react/', '/examples/react/'))
```

Comment:
This will only work locally, not on gh-pages as there path will be `voorhoede.github.io/fastatic/examples/react/`. Not strictly issue for this PR but will be later on.